src/app/rundown-view/components/rundown-header-context-menu/rundown-header-context-menu.component.ts

```diff
@@ -41,6 +42,10 @@ export class RundownHeaderContextMenuComponent {
       label: 'Reset Rundown',
       contextAction: (): void => this.openResetRundownDialog(),
     },
+    {
+      label: 'Reingest data',
+      contextAction: (): void => this.reingestData(),
```

Comment:
Instead of calling it reingestData, won't `reingestRundown` be more accurate?
Same comment for the label above.

Reply:
This was the label that Jacob decided on since it should also signify that you load in configuration changes. Is at least what is written on the task. https://tv2cms.atlassian.net/jira/software/c/projects/SOF/boards/784?selectedIssue=SOF-1665